doc/en/user/source/services/wps/processes/gs.rst

```diff
@@ -10,67 +10,345 @@ As with the "geo" processes, the names and definitions of these processes are su
+          <wps:LiteralData>Min</wps:LiteralData>
+        </wps:Data>
+      </wps:Input>
+      <wps:Input>
+        <ows:Identifier>function</ows:Identifier>
+        <wps:Data>
+          <wps:LiteralData>Max</wps:LiteralData>
+        </wps:Data>
+      </wps:Input>
+      <wps:Input>
+        <ows:Identifier>singlePass</ows:Identifier>
+        <wps:Data>
+          <wps:LiteralData>false</wps:LiteralData>
+        </wps:Data>
+      </wps:Input>
+    </wps:DataInputs>
+    <wps:ResponseForm>
+      <wps:RawDataOutput mimeType="application/json">
+        <ows:Identifier>result</ows:Identifier>
+      </wps:RawDataOutput>
+    </wps:ResponseForm>
+  </wps:Execute>
 
-JSON:
+The result:
 
 .. code-block:: json
 
-   {
-     "AggregationFunctions": ["Max"],
-     "AggregationAttribute": "energy_consumption",
-     "GroupByAttributes": [],
-     "AggregationResults": [
-       [500]
-     ]
-   }
+  {
+    "AggregationAttribute": "PERSONS",
+    "AggregationFunctions": ["Max", "Min", "Average", "Sum", "Count"],
+    "GroupByAttributes": [],
+    "AggregationResults": [
+      [29760021, 453588, 5038397.020408163, 246881454, 49]
+    ]
+  }
+
+The value of `AggregationResults` attribute should be read in a tabular way. The group by attributes come first in the order they appear in `GroupByAttributes` attribute. After comes the result of the aggregation functions in the order they appear in the `AggregationFunctions` attribute.
+
+This result should be interpreted like this:
+
+.. list-table::
+   :widths: 20 20 20 20 20
+
+   * - **Max**
+     - **Min**
+     - **Average**
+     - **Sum**
+     - **Count**
+   * - 29760021
+     - 453588
+     - 5038397.020408163
+     - 246881454
+     - 49
+
+To obtain the result in the XML format the request `wps:ResponseForm` element needs to be changed to:
+
+.. code-block:: xml
 
-WPS aggregate request WITH group by attributes:
+  <wps:ResponseForm>
+    <wps:RawDataOutput mimeType="text/xml">
+      <ows:Identifier>result</ows:Identifier>
+    </wps:RawDataOutput>
+  </wps:ResponseForm>
 
-XML:
+The result in XML format:
 
 .. code-block:: xml
 
-   <?xml version="1.0" encoding="UTF-8"?>
-   <AggregationResults>
-       <GroupByResult>
-           <object-array>
-               <string>FABRIC</string>
-               <big-decimal>500.0</big-decimal>
-           </object-array>
-           <object-array>
-               <string>HOUSE</string>
-               <big-decimal>6.0</big-decimal>
-           </object-array>
-           <object-array>
-               <string>SCHOOL</string>
-               <big-decimal>60.0</big-decimal>
-           </object-array>
-       </GroupByResult>
-   </AggregationResults>
-
-JSON:
+  <?xml version="1.0" encoding="UTF-8"?>
+  <AggregationResults>
+    <Min>453588.0</Min>
+    <Max>2.9760021E7</Max>
+    <Average>5038397.020408163</Average>
+    <Sum>2.46881454E8</Sum>
+    <Count>49</Count>
+  </AggregationResults>
+
+
+===============
+ Example 2
+===============
```

Comment:
Think this is the "wrong" header based on sphinx conventions - trying to keep things consistent across the different files.

Since the top of the file has:
```
 GeoServer processes
 ===================
```

This would show up in the sphinx table of contents as a new page!

Consider using::

      Aggregate GroupBy Example
      ```````````````````````````